src/main/resources/application.yml

```diff
@@ -9,6 +9,56 @@ spring:
   threads:
     virtual:
       enabled: "\${VIRTUAL_THREADS_ENABLED:true}"
+  cloud:
+    function:
+      definition: registryProducer
+    stream:
+      binders:
+        kafka-registry-producer:
+          type: kafka
+          environment:
+            spring.cloud.stream.kafka.binder:
+              brokers: "\${KAFKA_BINDER_BROKER:}"
+              configuration:
+                sasl.jaas.config: "\${KAFKA_REGISTRY_PRODUCER_SASL_JAAS_CONFIG:}"
```

Comment:
let's create a PR on aks-deployment to set kafka variables as done for pu-sil:
https://github.com/pagopa/p4pa-payhub-deploy-aks/blob/main/helm/_global/p4pa-pu-sil.yaml#L57
https://github.com/pagopa/p4pa-payhub-deploy-aks/blob/main/helm/_global/p4pa-pu-sil.yaml#L80